confidential-assets/src/confidentialNormalization.ts

```diff
@@ -48,8 +50,12 @@ export class ConfidentialNormalization {
     this.decryptionKey = args.decryptionKey;
     this.unnormalizedEncryptedBalance = args.unnormalizedEncryptedBalance;
     this.balanceAmount = args.balanceAmount;
-    this.normalizedConfidentialAmount = args.normalizedConfidentialAmount;
+    this.normalizedConfidentialBalance = args.normalizedConfidentialAmount;
     this.randomness = args.randomness;
+    this.normalizedEncryptedBalance = this.normalizedConfidentialBalance.getAmountEncrypted(
```

Comment:
I'm assuming the user will always want this, so it's good to compute it now, rather than doing it lazily later?

Reply:
Not sure if you are asking a question.

Am I lazily doing it later in this PR?  I'm doing it 'now' in the constructor right?

Reply:
I see that. Let me rephrase, why are we computing this now vs doing it when the user actually wants it.

Reply:
Yes, because the user will always want this if they are going to normalize

Reply:
okay cool cool